sandbox/component/button/script.es

```diff
@@ -0,0 +1,19 @@
+import '../../../cwel/script/main';
```

Comment:
@NathanRodger I would ditch this file if it's already in the `~/cwel/component/button` directory.